src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,47 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int LOGIN_MINIMAL_LENGTH = 2;
+    private static final int PASSWORD_MINIMAL_LENGTH = 6;
+    private static final int MINIMAL_AGE = 18;
+    private static final int MAXIMUM_AGE = 123;
```

Comment:
checking max age is redundant

Reply:
I deleted this and all checks for maximum age. But question is - in real-life projects we don't need to check cases with the maximum age?